configmaptocrs/main.go

```diff
@@ -44,7 +45,8 @@ func main() {
 
 	f, err := os.Create(filepath.Join(inputDirPath, outPutFileName))
 	if err != nil {
-		log.Fatalf("failed to create output file: %s", err)
+		log.Println("failed to create output file, writing to stdout")
```

Comment:
I'd be explicit about our intentions, and provide a parameter to drive the behaviour, otherwise this will hide errors (i.e. the user thinks he set a file)

Reply:
done. 
--output flag added